recipe/meta.yaml

```diff
@@ -12,12 +12,12 @@
 #       A Public Release for ISIS3.6.1:                        {% set version = "3.6.1" %}
 #       A Release Candidate for ISIS3.6.1:                     {% set version = "3.6.1_RC" %}
 #       A custom build of ISIS3.6.1 for the CaSSIS mission:    {% set version = "3.6.1_cassis" %}
-{% set version = "3.6.1" %}
+{% set version = "3.7.0_RC" %}
 
 # This is the branch that conda build will pull from the ISIS repository. If you are building for a
 # general public release, leave the branch as "release". If you are building a custom build, like
 # for a mission for example, simply change the value to the branch of choice.
-{% set git_branch = "release" %}
+{% set git_branch = "dev" %}
```

Comment:
Should this be a branch or should this be a tag?

Reply:
We should be doing the build off of a release tarball

Reply:
I guess I'm confused about the question here...?

Reply:
I'm even more confused. I can't even find a reference to `git_branch` in the docs https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#source-section.

But it also looks like `git_tag` exists, in which case we could use this 3.7.0_RC tag to do the build, which seems correct to me? https://stackoverflow.com/questions/22546878/possible-to-do-conda-build-from-github-branch

Reply:
@SgStapleton @kberryUSGS For the RCs, using the tag makes sense. For the full 3.7.0 release we should be creating a release tarball on github. Then, for the release build we should be using that tarball.

See how we build [the csm on conda-forge](https://github.com/conda-forge/csm-feedstock/blob/master/recipe/meta.yaml#L8).

Reply:
I guess the main concern I have here is when there needs to be a special build for a particular mission. They will need to either pull from a particular branch, or they will need to create a tag for their build in the repository. The latter would probably be preferred to keep track of the snapshots of the repo at particular builds, but I just wanted to make sure this makes sense to everyone.